src/Compilers/VisualBasic/Portable/Symbols/ConstraintsHelper.vb

```diff
@@ -1110,28 +1094,61 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return New CompoundDiagnosticInfo(diagnostics)
         End Function
 
+        Public Enum ConstructorConstraintError
+            None
+            NoPublicParameterlessConstructor
+            MustInheritType
+            HasRequiredMembers
+        End Enum
+
         ''' <summary>
         ''' Return true if the class type has a public parameterless constructor.
         ''' </summary>
```

Comment:
>constructor.ContainingType [](http://example.com/codeflow?start=34&length=26)

```type```?

Reply:
It looks like codeflow messed this comment up, I'm not sure what it was supposed to be on, but I see it on the Function header of `HasPublicParameterlessConstructor`

Reply:
> It looks like codeflow messed this comment up, I'm not sure what it was supposed to be on

I think that comment was about ```constructor.ContainingType``` in 
```TypeOf constructor.ContainingType Is SourceMemberContainerTypeSymbol``` condition below. The condition is gone now.